README.md

```diff
@@ -145,6 +147,42 @@ pk.search('Paris', {
 | `query` | `string` | Search terms |
 | `opts` | `key-value mapping` (optional) | Search-specific parameters (see [options](#pkoptions)) |
 
+### `pk.reverse()`
+
+Performs a reverse geocoding search and returns a Promise, which response is a list of results alongside some request metadata.
+The options passed as second parameter override the global parameters only for the current query.
+Any `coordinates` previously set as option would be overriden by the coordinates passed as first argument.
+
+```js
+pk.reverse('48.871086,2.3036339', {
+  countries: ['fr'],
+  maxResults: 5,
+}).then((res) => {
+  console.log(res.results);
+});
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `coordinates` | `string` | `"lat,lng"` formatted coordinates. |
+| `opts` | `key-value mapping` (optional) | Search-specific parameters (see [options](#pkoptions)) |
+
+**Note:** when calling `pk.reverse()`, the API automatically sets `countryByIP` to `true`. Explicitely set it to `false` to turn it off.
```

Comment:
+ and specify the country in `countries`

Reply:
This I explain later. Dev doesn't have to set it here, it could be set as a global parameter. I still add the `countries` option in all my examples.